### DISCUSSION TITLE
Macro within a Tap Dance

### DISCUSSION THREAD

**Discussion by Railang** [KinesisCorporation/Adv360-Pro-ZMK#121] [Q&A]
So I have a Macro that is known working, however when I put it into a Tapdance sequence it no longer works.

macro_copy: macro_copy{
compatible = "zmk,behavior-macro";
label = "macro_copy";
#binding-cells = <0>;
bindings = <&macro_press &kp LCTRL>
, <&macro_press &kp C>
, <&macro_release &kp C>
, <&macro_release &kp LCTRL>;
};

    td0: tap_dance_0 {
            compatible = "zmk,behavior-tap-dance";
            label = "TAP_DANCE_0";
            #binding-cells = <0>;
            tapping-term-ms = <100>;
            bindings = <&kp C>, <&macro_copy>;
        };

bindings = <
&td0

**Comment by ReFil** [comment]
I don't know of any specific reason why macro in tapdance doesn't work, i will investigate and get back to you soon
